### PR TITLE
Fixes #34098 - correctly print proxy type in Puppet message

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/katello/katello/hostname-change.rb [hunk 1]
```diff
@@ -156,7 +156,7 @@ def next_steps_message
   #{@options[:program]}-installer --foreman-proxy-foreman-base-url https://#{@new_hostname} \\
                                   --foreman-proxy-trusted-hosts #{@new_hostname}
 
-  If Puppet is enabled on the #{proxy}, add the following to the installer command:
+  If Puppet is enabled on the #{@proxy}, add the following to the installer command:
```
Comment (evgeni) [inline packages/katello/katello/hostname-change.rb:159]: this is the real fix, the rest is "oh god, why did I look"

File: packages/katello/katello/helper.rb [hunk 2]
```diff
@@ -32,8 +32,16 @@ def error_message
       "This utility can't run on a non-katello system."
     end
 
+    def scenario_yaml(scenario)
+      File.join(scenarios_path, "#{scenario}.yaml")
+    end
```
Comment (ekohl) [inline packages/katello/katello/helper.rb:37]: Nit: Above are a bunch of paths. Should this be together with `last_scenario_yaml` (which should also use `File.join()` IMHO, but we're getting deep into to "why did I look" territory).?

File: packages/katello/katello/helper.rb [hunk 3]
```diff
@@ -32,8 +32,16 @@ def error_message
...
+
     def load_scenario_data(scenario)
-      YAML.load_file(File.join(scenarios_path, "#{scenario}.yaml"))
+      if File.exist?(scenario_yaml(scenario))
+        YAML.load_file(scenario_yaml(scenario))
+      else
+        {}
```
Comment (ekohl) [inline packages/katello/katello/helper.rb:43]: Looks like `load_scenario_answers` can also deal with `nil` so it can be
```suggestion
```

File: packages/katello/katello/katello.spec [hunk 4]
```diff
@@ -132,6 +132,9 @@ Provides a federation of katello services
 # the files section is empty, but without it no RPM will be generated
 
 %changelog
+* Tue Dec 07 2021 Evgeni Golov - 4.4.0-0.2.master
+- Fixes #34098 - correctly print proxy type in Puppet message
```
Comment (ekohl) [inline packages/katello/katello/katello.spec:136]: Should this be squashed into another commit?

**Review by ekohl — COMMENTED** [review]
Thanks!

File: packages/katello/katello/helper.rb (hunk 2 above)
Comment (evgeni) [reply packages/katello/katello/helper.rb:37]: Can certainly move it up there.

**Review by ekohl — APPROVED** [review]
